### ISSUE TITLE
Table contents of "Documentation" misses a lot of pages on Xtext home page

### ISSUE THREAD

**Issue by scharf** [eclipse/xtext#1095]
I was searching for the "seven languages xtext" and [google found it](https://eclipse.org/Xtext/documentation/201_sevenlang_introduction.html), but I wonder why this is not part of the documentation index.

Looking into the HTML for the table of contents, it seems that  quite a few parts are commented out:

Is that intentional or by accident?

```html
<ul id="nav-outline" style="margin-left: 0px;">
	<li class="nav-part">Getting Started</li>
	<!--li><a href="101_five_minutes.html">5 Minutes Tutorial</a>
	</li-->
	<li><a href="102_domainmodelwalkthrough.html">15 Minutes Tutorial</a>
	</li>
	<li><a href="103_domainmodelnextsteps.html">15 Minutes Tutorial - Extended</a>
	</li>
	<li><a href="104_jvmdomainmodel.html">Five simple steps to your JVM language</a>
	</li>
	<!--li class="nav-part">Seven JVM Languages Built With Xbase</li>
	<li><a href="201_sevenlang_introduction.html">Introduction</a>
	</li>
	<li><a href="202_scripting.html">Scripting Language</a>
	</li>
	<li><a href="203_builddsl.html">Build Language</a>
	</li>
	<li><a href="204_mongodb.html">DSL for MongoDB</a>
	</li>
	<li><a href="205_guice.html">DSL for Guice</a>
	</li>
	<li><a href="206_httprouting.html">Http Routing Language</a>
	</li>
	<li><a href="207_template.html">Template Language</a>
	</li>
	<li><a href="208_tortoise.html">Little Tortoise</a>
	</li-->
	<li class="nav-part">Reference Documentation</li>
	<li><a href="301_grammarlanguage.html">The Grammar Language</a>
	</li>
	<li><a href="302_configuration.html">Configuration</a>
	</li>
	<li><a href="303_runtime_concepts.html">Language Implementation</a>
	</li>
	<li><a href="305_xbase.html">Integration with Java</a>
	</li>
	<!--li><a href="306_mwe2.html">MWE2</a>
	</li-->
	<!--li><a href="307_special_languages.html">Typical Language Configurations</a>
	</li-->
	<li><a href="308_emf_integration.html">Integration with EMF</a>
	</li>
	<li><a href="310_eclipse_support.html">Eclipse Support</a>
	</li>
	<!--li><a href="320_idea_support.html">IntelliJ IDEA Support</a>
	</li-->
	<li><a href="330_web_support.html">Web Editor Support</a>
	</li>
	<li><a href="350_continuous_integration.html">Continuous Integration</a>
	</li>
	<!--li class="nav-part">Appendix</li>
	<li><a href="401_migrating_from_1_0_x.html">Migrating from Xtext 1.0.x to 2.0</a>
	</li>
	<li><a href="402_migrating_from_0_7.html">Migrating from Xtext 0.7.x to 1.0</a>
	</li-->
	<li class="nav-part">Additional Resources
	</li><li><a href="http://download.eclipse.org/modeling/tmf/xtext/javadoc/2.9/">API Documentation (JavaDoc)</a>
</li></ul>
```

**Comment by spoenemann** [comment]
It is intentional. The plan was to move those parts to another place, e.g. https://github.com/xtext/seven-languages-xtext, but no one has taken care of that yet. Let's leave this issue open as a reminder.

**Comment by scharf** [comment]
This is unfortunate. I'd suggest to uncomment it until that happend. I found it, because I knew the seven languages exist. But new users will not find this excellent resource. 

I added a PR for seven-languages-xtext https://github.com/xtext/seven-languages-xtext/pull/7 to point to the "hidden" documentation gem.

**Comment by spoenemann** [comment]
I moved the Seven Languages documentation to https://github.com/xtext/seven-languages-xtext/tree/master/documentation

What is left to do: Move [306_mwe2.md](https://github.com/eclipse/xtext/blob/website-master/xtext-website/documentation/306_mwe2.md) to a documentation page of MWE2, if appropriate. Who could take care of that?

**Comment by miklossy** [comment]
- [ ] The MWE2 documentation has been added to the MWE2 GitHub repository.
- [ ] What is left over is to remove the [306_mwe2.md](https://github.com/eclipse/xtext/blob/website-published/xtext-website/documentation/306_mwe2.md) file from the Xtext repository.

**Comment by miklossy** [comment]
Fixed in 2.22
